### PR TITLE
docs: add install Firebase CLI step before using its commands

### PR DESCRIPTION
**Pull request by nhannqgh**
You need to install Firebase CLI before using `firebase login` and `firebase init` commands.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

There is no step to install Firebase CLI before using its commands.

## What is the new behavior?

Add install Firebase CLI step before using its commands.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by nhaancs** [comment]
@googlebot I signed it!

File: aio/content/start/start-deployment.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ One of the easiest ways to get your site live is to host it using Firebase.
 1. Sign up for a firebase account on [Firebase](https://firebase.google.com/ "Firebase web site").
 1. Create a new project, giving it any name you like.
 1. Add the `@angular/fire` schematics that will handle your deployment using `ng add @angular/fire`.
+1. Install Firebase CLI globaly using `npm install -g firebase-tools`.
```
Comment (kapunahelewong) [inline aio/content/start/start-deployment.md:66]:
```suggestion
1. Install [Firebase CLI](https://firebase.google.com/docs/cli) globally using `npm install -g firebase-tools`. 
```

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
This is a nice update. Thank you, @nhaancs! Just two small suggestions, one is a link to the Firebase CLI documentation and the other is just a typo fix.

File: aio/content/start/start-deployment.md (hunk 1 above)
Comment (nhaancs) [reply aio/content/start/start-deployment.md]: @kapunahelewong thanks for your suggestions, I added your updates.

**Comment by kapunahelewong** [comment]
You're fast! 🏎 Could you squash and rebase? Here are [my notes on both](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-rebase) if you need them. Let me know if you need any help!

**Comment by nhaancs** [comment]
Hi @kapunahelewong , I did squash and rebase. There were a conflict and I fixed it. It seems that I did something wrong, there are many files changed listed here. What should I do next, thank you.

**Review by kapunahelewong — APPROVED** [review]
I might have fixed it, @nhaancs. I'll keep watch to see if the tests pass.

Reviewed-for: required-minimum-review

**Review by kapunahelewong — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Review by aikithoughts — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Pull request closed without merging**